### ISSUE TITLE
some issues with x-tile

### ISSUE THREAD

**Issue by wisdomlight** [giuspen/x-tile#15]
Hi, are you still developing x-tile?

attached is a screen shot to demonstrate my points. (Debian 10; GNOME 3.38.1)

I have 2 questions.

1. x tile does not recognize the Cherry Tree window -(ironic)

2. Why are there two rows of vertical tick boxes next to each window in
the list.

Thank you

Best wishes
![x-tile-Screenshot from 2020-11-27 15-10-00](https://user-images.githubusercontent.com/14957169/100489539-9b4fd100-310c-11eb-83f6-d6b5f598a404.png)

**Comment by giuspen** [comment]
Hi,
I'm trying to keep it up, I'm not planning to implement new stuff but do bugfixing.
The two columns of checkboxes are for the target monitor, when it detects that there are two monitors.
It seems the old version of cherrytree (gtk2), can you try with the gtk3 version? (0.99.x)

**Comment by novaTopFlex** [comment]
Yes, I believe that my Gtk+3.0 applications are compatible with `x-tile` and my GTK 4.0 applications are the same way. Applications with Gtk+2.0 or Qt4 may not successfully be recognized and/or tiled properly.
